src/smc-util/message.js

```diff
@@ -554,6 +554,7 @@ message({
   hub: required, // ip address (on vpn) of hub user connected to.
   account_id: required, // uuid of user's account
   email_address: undefined, // email address they signed in under
+  lti_id: undefined, // alternatively if email_address isn't set, there must be an lti_id
```

Comment:
I'm trying to figure out how your broke anonymous account creation, but at least this comment is a hint.  An anonymous account definitely doesn't have an email address or lti_id set...